## Practice Exercise 1

You are provided with 2 lists that contain the data of an ecommerce website. The first list contains the data for the number of items sold for a particular product and the second list contains the price of the product sold. As a part of this exercise, solve the questions that are provided below.

In [3]:
number = [8, 9, 9, 1, 6, 9, 5, 7, 3, 9, 7, 3, 4, 8, 3, 5, 8, 4, 8, 7, 5, 7, 3, 6, 1, 2, 7, 4, 7, 7, 8, 4, 3, 4, 2, 2, 2, 7, 3, 5, 6, 1, 1, 3, 2, 1, 1, 7, 7, 1, 4, 4, 5, 6, 1, 2, 7, 4, 5, 8, 1, 4, 8, 6, 2, 4, 3, 7, 3, 6, 2, 3, 3, 3, 2, 4, 6, 8, 9, 3, 9, 3, 1, 8, 6, 6, 3, 3, 9, 4, 6, 4, 9, 6, 7, 1, 2, 8, 7, 8, 1, 4]
price = [195, 225, 150, 150, 90, 60, 75, 255, 270, 225, 135, 195, 30, 15, 210, 105, 15, 30, 180, 60, 165, 60, 45, 225, 180, 90, 30, 210, 150, 15, 270, 60, 210, 180, 60, 225, 150, 150, 120, 195, 75, 240, 60, 45, 30, 180, 240, 285, 135, 165, 180, 240, 60, 105, 165, 240, 120, 45, 120, 165, 285, 225, 90, 105, 225, 45, 45, 45, 75, 180, 90, 240, 30, 30, 60, 135, 180, 15, 255, 180, 270, 135, 105, 135, 210, 180, 135, 195, 225, 75, 225, 15, 240, 60, 15, 180, 255, 90, 15, 150, 230, 150]

#### How many different products are sold by the company in total?

-  99
-  100
-  101
-  102


In [18]:
import numpy as np
aq=np.array(price)
sum=np.mean(aq)
print(sum)

# Type your code here

139.01960784313727
285


#### How many items were sold in total?

-  460
-  490
-  500
-  520


In [ ]:
# Type your code here
aq=np.array(numbers)

sum=np.sum(numbers)
print(sum)

#### What is the average price of the products sold by the ecommerce company?

-  139
-  151
-  142
-  128

In [ ]:
# Type your code here
sum=np.mean(aq)
print(sum)

#### What is the price of the costliest item sold?

-  225
-  310
-  280
-  285

In [ ]:
# Type your code here
max=np.max(price)
print(max)

#### What is the total revenue of the company? [Revenue = Price\*Quantity]

-  67100
-  53900
-  45300
-  71200

In [22]:
# Type your code here
p1=np.array(price)
n1=np.array(number)
it=p1*n1
print(np.sum(it))


67100


#### Demand for the 20th product in the list is more than the 50th product. [True/False]
-  True
-  False
-  Can't be calculated

In [26]:
t1=p1[21]
print(t1>p1[51])


False


#### How many products fall under the category of expensive goods? 
An expensive good is that good whose price is more than the average price of the products sold by the company.

-  48
-  50
-  52
-  54

In [32]:
# Type your code here
a1=np.average(p1)
f1=p1[p1>a1]
print(len(f1))

52


In [34]:
!git status
pwd

On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	../.PyCharm2018.3/
	../.PyCharmCE2019.1/
	../.PyCharmCE2019.3/
	../.angular-config.json
	../.atom/
	../.aws/
	../.config/
	../.gitconfig
	../.ipynb_checkpoints/
	../.ipython/
	../.jupyter/
	../.matplotlib/
	../.node_repl_history
	../.pylint.d/
	../.scala_history
	../.spyder-py3/
	../.ssh/
	../.vscode/
	../27th sep2020.ipynb
	../28th september.ipynb
	../3D Objects/
	../AppData/
	../BullseyeCoverageError.txt
	../Contacts/
	../Desktop/
	../Documents/
	./
	../Favorites/
	../Github/
	../Intel/
	../IntelGraphicsProfiles/
	../Links/
	../MicrosoftEdgeBackups/
	../Music/
	../NTUSER.DAT
	../NTUSER.DAT{8ebe95f6-3dcb-11e8-a9d9-7cfe90913f50}.TxR.blf
	../NTUSER.DAT{8ebe95f7-3dcb-11e8-a9d9-7cfe90913f50}.TM.blf
	../NTUSER.DAT{8ebe95f7-3dcb-11e8-a9d9-7cfe90913f50}.TMContainer00000000000000000001.regtrans-ms
	../NTUSER.DAT{8ebe95f7-3dcb-11e8-a9d9-7cfe90913f50}.TMContainer0000000000000000

NameError: name 'pwd' is not defined

	../codecommit/
	../java_error_in_pycharm_11024.log
	../java_error_in_pycharm_2400.log
	../java_error_in_pycharm_5028.log
	../java_error_in_pycharm_5100.log
	../java_error_in_pycharm_5872.log
	../java_error_in_pycharm_5972.log
	../java_error_in_pycharm_7116.log
	../new 1
	../ntuser.dat.LOG1
	../ntuser.dat.LOG2
	../ntuser.ini
	../opera autoupdate/
	../reactpoc/

nothing added to commit but untracked files present (use "git add" to track)


In [41]:
!git clone https://github.com/pratik217/datascience_pys

Cloning into 'datascience_py'...


In [63]:
cd C:/Users/Priyanka/Downloads/ds1/

[master (root-commit) ae2e62a] first1numy
 1 file changed, 405 insertions(+)
 create mode 100644 03_Practice+Exercise+1.ipynb


To https://github.com/pratik217/datascience_py
 * [new branch]      master -> master
